lxd/instances_get.go

```diff
@@ -427,8 +428,11 @@ func doInstancesGet(d *Daemon, r *http.Request) (interface{}, error) {
 				} else if strings.HasPrefix(mux.CurrentRoute(r).GetName(), "vm") {
 					instancePath = "virtual-machines"
 				}
-				url := fmt.Sprintf("/%s/%s/%s", version.APIVersion, instancePath, projectInstance[1])
-				resultString = append(resultString, url)
+				url := api.NewURL().Path(version.APIVersion, instancePath, projectInstance[1])
+				if projectInstance[0] != project.Default {
```

Comment:
No need for this logic as it is built in to:

https://github.com/lxc/lxd/blob/master/shared/api/url.go#L48-L56